plugins/modules/locale_gen.py

```diff
@@ -174,13 +182,13 @@ def apply_change_ubuntu(targetState, name):
             f.close()
         # Purge locales and regenerate.
         # Please provide a patch if you know how to avoid regenerating the locales to keep!
-        localeGenExitValue = call(["locale-gen", "--purge"])
-
-    if localeGenExitValue != 0:
-        raise EnvironmentError(localeGenExitValue, "locale.gen failed to execute, it returned " + str(localeGenExitValue))
+        with runner("purge") as ctx:
+            ctx.run()
 
 
 def main():
+    global module
```

Comment:
How much work would it be to get rid of that global variable completely?

Reply:
going to make this a MH module, but going in small bites ;-) - the global will disappear soon